cli/azd/cmd/testdata/TestUsage-azd-infra-synth.snap

```diff
@@ -7,6 +7,7 @@ Usage
 Flags
     -e, --environment string 	: The name of the environment to use.
         --force              	: Overwrite any existing files without prompting
+    -o, --output string      	: The output format (the supported formats are none).
```

Comment:
This is a negative UX impact -- we're offering a flag that doesn't change the behavior of the command.